CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 6.4.2 / 2023-12-07
+
+## What's Changed
+* Release/6.1.0 by @ifrit98 in https://github.com/opentensor/bittensor/pull/1550
+* Release/6.2.0 by @ifrit98 in https://github.com/opentensor/bittensor/pull/1567
+* Release/6.3.0 by @ifrit98 in https://github.com/opentensor/bittensor/pull/1582
+* Release/6.4.0 by @ifrit98 in https://github.com/opentensor/bittensor/pull/1599
+* revert subtensor local to finney by @ifrit98 in https://github.com/opentensor/bittensor/pull/1600
```

Comment:
can we have some meaningful sentences for this changes? It can be concise

what do you think about that?